apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -1,14 +1,18 @@
 use serialport::{self, SerialPort};
-use std::{fmt::Display, io, thread, time::Duration};
+use std::{
+    io, thread,
+    time::{Duration, Instant},
+};
 use uinput::{event::keyboard, Device};
 
-const POLL_INTERVAL_MS: u64 = 10;
+const POLL_INTERVAL: Duration = Duration::from_millis(10);
+const MAX_ECHO_RESPONSE_WAIT: Duration = Duration::from_secs(1);
```

Comment:
Rust has types for representing durations of time. This ensures we can't mess up the units.